src/Tasks/GenerateManifestBase.cs

```diff
@@ -485,6 +483,8 @@ private bool InitializeManifest(Type manifestType)
             {
                 _manifest.Description = Description;
             }
+            _manifest.LauncherBasedDeployment = LauncherBasedDeployment;
+            _manifest.AssemblyName = AssemblyName;
```

Comment:
I gather this AssemblyName is just the file name, i.e., apphost.exe rather than a full path, right?

Reply:
https://github.com/dotnet/msbuild/pull/5745#discussion_r492208787

Reply:
Correct, it will be the targetname e.g. consoleapp1.exe.